### ISSUE TITLE
Production Deployment Failure

### ISSUE THREAD

**Issue by AndreaBeth11** [clifford-fra/GoogleRecaptchav2#2]
I got everything running as expected in Sandbox.  When I go to install all the components in Prod, I get this error:

`Deployment CompleteTest failure, method: CaseCommProcessorTest.test -- System.DmlException: Insert failed. First exception on row 0; first error: FIELD_INTEGRITY_EXCEPTION, field integrity exception (Guest users cannot be record owners.): [] stack Class.CaseCommProcessorTest.setup: line 11, column 1`

Which is sort of funny because I'm using the ReCaptcha in a Flow in our Community to prevent a Guest user from owning a case.

**Comment by clifford-fra** [comment]
Hi Andrea,

this seems like a community flow error. Recently, guest user can't be record owners.

This might be of help:
https://help.salesforce.com/articleView?id=networks_assign_guest_records.htm&type=5

see also:
https://releasenotes.docs.salesforce.com/en-us/spring20/release-notes/rn_forcecom_flow_fbuilder_system_mode.htm

**Comment by AndreaBeth11** [comment]
Yeah, the guest user issue is the reason the Global Action I had stopped working and why I'm using a Flow and need the Recaptcha.  My default is already set & I can't set the Flow context because I can't changeset the Flow from Sandbox without Recaptcha components being installed.  I think I've found that space between a rock and a hard place.
